internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,43 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	dataSlice := strings.Split(data, ",") // Получение количества шагов и времени из input строки
+	if len(dataSlice) != 2 {
+		return 0, 0, fmt.Errorf("Неверный формат данных")
+	}
+
+	steps, err := strconv.Atoi(dataSlice[0]) // Конвертация типа шагов
+	if err != nil || steps <= 0 {
+		return 0, 0, fmt.Errorf("Something went wrong")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```